wipe onboarding server for makers in bad standing (#1106)
helium/HIP: 0000-remove-cn470-region-from-poc.md

> # HIP XXXX: Remove CN470 from PoC

Reviewer: ```suggestion
# HIP XXXX: Remove CN470 from IOT PoC
```

> - Author(s): [@bfgneil](https://github.com/bfgneil)

Reviewer: ```suggestion
- Author: [@BFGNeil](https://github.com/BFGNeil)
```

- Start Date: 2024-09-19
- Category: Economic
- Original HIP PR:
- Tracking Issue:

> - Voting Requirements: veIOT holders

Reviewer: ```suggestion
- Voting Requirements: veIOT Holders
```

## Summary

> This HIP proposes to remove the CN470 region from earning poc until a time when classifiers and other methods of anti gaming can identify and stop gaming.

Reviewer: ```suggestion
This proposal removes the CN470 region from earning Proof-of-Coverage rewards on the IOT network until such a time when classifiers and other methods of anti-gaming can identify and stop gaming in this region.
```

## Motivation

> Currently around 14% of PoC rewards are going to hotspots in the cn470 region.
> 
> Whilst the gaming is easy to detect by eye, its hard to detect it programatically without finding caveats.
> 
> The closest we have is the newer ITM model, this can be bypassed by setting height on hotspots and would need further development and time to implement.
> 
> These hotspots represent under 1,000 of DC usage.

Reviewer: ```suggestion
Whilst the gaming is easy to detect by eye, its hard to detect it programmatically without finding caveats. The closest we have is the newer ITM model, this can be bypassed by setting height on Hotspots and would need further development and time to implement.

Currently around 14% of PoC rewards are going to Hotspots in the CN470 region. These Hotspots represent under 1,000 of DC usage.
```

rewrite for clarity. also 1K DC per ____?

## Stakeholders

All customers (e.g., sensor owners), hosts, and operators of the Helium IOT Network.

## Detailed Explanation

> We propose to remove the CN470 region from PoC reward calculations.
> 
> We then propose a new denylist focused on any hotspots moving from this region to others be added.
> 
> We also propose these changes be removed when anti gaming techniques have been developed to adress these issues.

Reviewer: ```suggestion
We propose to remove the CN470 region from PoC reward calculations. We then propose a new denylist focused on any Hotspots moving from this region to others be added. We also propose these changes be removed when anti gaming techniques have been developed to address these issues.

This proposal does not remove Data Transfer capability for CN470 Hotspots.
```

## Implementation

> - Remove the cn470 region from PoC calculations
> - Track any hotspots in the cn470 region for location assertions out of region and produce a list to be added to the denylist

Reviewer: ```suggestion
- Remove the CN470 region from PoC calculations
- Track any Hotspots in the CN470 region for location assertions out of region and produce a list to be added to the denylist
```

## Alternatives

> - Let gaming happen forever

Reviewer: ```suggestion
- Do nothing and potentially let gaming happen forever
```

## Drawbacks

- None

## Unresolved Questions

-- none so far

## Deployment Impact

> - less gaming.

Reviewer: ```suggestion
We expect less gaming on the IOT network as a whole with this change and more PoC rewards going to other regions.
```
